HotelPriceScout/Pages/Settings.razor

```diff
@@ -0,0 +1,303 @@
+@page "/settings"
+
+
+
+<h1>Settings</h1>
+<br />
+<SfDropDownList TItem="Roomtypes" TValue="string" DataSource="Rooms" Placeholder="Select room type" PopupWidth="300px" PopupHeight="200px" Width="300px">
+	<DropDownListFieldSettings Text="Roomtype" Value="RoomValue">
+	</DropDownListFieldSettings>
+</SfDropDownList>
+<br />&nbsp;
+<br />
+<br />&nbsp;
+<br />
+
+<SfMultiSelect TValue="string" TItem="Hotels" AllowFiltering="true" Placeholder="Select hotels" Mode="@VisualMode.CheckBox" DataSource="@Hotellist" PopupWidth="300px" PopupHeight="200px" Width="300px">
+	<MultiSelectFieldSettings Text="HotelName" Value="Zipcode">
+	</MultiSelectFieldSettings>
+</SfMultiSelect>
+<br />&nbsp;
+<br />
+<br />&nbsp;
+<br />
```

Comment:
Disse skal flyttes til dashboarden på en eller anden måde. Kommenter det ud for nu så det kan bruges senere til dashboard